shared-bindings/displayio/Group.c

```diff
@@ -215,9 +215,14 @@ MP_DEFINE_CONST_FUN_OBJ_2(displayio_group_append_obj, displayio_group_obj_append
 //|
 STATIC mp_obj_t displayio_group_obj_insert(mp_obj_t self_in, mp_obj_t index_obj, mp_obj_t layer) {
     displayio_group_t *self = native_group(self_in);
-    size_t index = mp_get_index(&displayio_group_type, common_hal_displayio_group_get_len(self), index_obj, false);
-    common_hal_displayio_group_insert(self, index, layer);
-    return mp_const_none;
+    if ((size_t) MP_OBJ_SMALL_INT_VALUE(index_obj) == common_hal_displayio_group_get_len(self)){
+        return displayio_group_obj_append(self_in, layer);
+    }
+    else {
+        size_t index = mp_get_index(&displayio_group_type, common_hal_displayio_group_get_len(self), index_obj, false);
+        common_hal_displayio_group_insert(self, index, layer);
+        return mp_const_none;
+    }
```

Comment:
Since the `if` returns, you can omit the `else`.

```suggestion
    size_t index = mp_get_index(&displayio_group_type, common_hal_displayio_group_get_len(self), index_obj, false);
    common_hal_displayio_group_insert(self, index, layer);
    return mp_const_none;
```

Reply:
Thanks @tannewt, I made these changes, verified them again and committed.  Also I deleted the unnecessary mpy-cross file.